doc/en/user/source/conf.py

```diff
@@ -215,17 +230,16 @@
   'fncychap': '\\usepackage[Sonny]{fncychap}',
 'preamble': #"""\\usepackage[parfill]{parskip}
   """
-	\\hypersetup{
-		colorlinks = true,
+    \\hypersetup{
```

Comment:
Is this comment, or just weird syntax? If its just a comment, can it be removed?

Reply:
tabs vs spaces